src/applications/disability-benefits/new-686/config/chapters/report-divorce/former-spouse-information/formerSpouseInformation.js

```diff
@@ -0,0 +1,81 @@
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import { validateName } from '../../../utilities';
+import { genericSchemas } from '../../../generic-schema';
+
+const {
+  fullName: formerSpouseName,
+  date: dateOfDivorce,
+  genericLocation: locationOfDivorce,
+} = genericSchemas;
+
+export const schema = {
+  type: 'object',
+  required: [
+    'formerSpouseName',
+    'dateOfDivorce',
+    'locationOfDivorce',
+    'isMarriageAnnulledOrVoid',
+  ],
+  properties: {
+    formerSpouseName,
+    dateOfDivorce,
+    locationOfDivorce,
+    isMarriageAnnulledOrVoid: { type: 'boolean' },
+    explanationOfAnnullmentOrVoid: {
+      type: 'string',
+      maxLength: 500,
+      pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*]+$',
```

Comment:
:confused: 
Does this...require that two or more consecutive whitespace characters exist.....?
Does this regex come from a downstream service's requirement? It smells like an EVSS regex :speak_no_evil: .

Reply:
I did lift it from another form that was meant to submit to evss. But you bring up an interesting thought: This one won't actually submit to evss...so I wonder if there's a better regex to use.